evm/src/cpu/gas.rs

```diff
@@ -113,6 +112,14 @@ fn eval_packed_accumulate<P: PackedField>(
     yield_constr.constraint_transition(
         lv.op.jumpdest_keccak_general * (gas_diff - jumpdest_keccak_general_gas_cost),
     );
+
+    // For PROVER_INPUT and PUSH operations.
+    // PUSH operations are differentiated from PROVER_INPUT by their 6th bit set to 1.
```

Comment:
Nit: I say 5th in that context. I don't know which convention has been used throughout the document, but we should harmonize.

Reply:
In the rest of the file, they use 6th in this context.